code/src/components/SecondQuestion.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import './secondQuestion.css'
+
+export const SecondQuestion = ({ answer, setAnswer }) => {
+
+    return (
+        <section className="second-question" id="answer">
+            <h1 tabIndex="0">What did, or could have made your day better?</h1>
+            <h2 tabIndex="0">Every little moment counts!</h2>
+            {/* <form onSubmit={event => event.preventDefault()} /> */}
+            <label tabIndex="0">
+                <input
+                    className="better-day"
+                    pattern="^[a-zA-Z ]*$"
+                    id="answer"
+                    type="text"
+                    value={answer}
+                    onChange={event => setAnswer(event.target.value)}
+                    minLength="10"
+                    placeholder="Be kind to yourself"
+                    required
+                />
+            </label>
+            <button><a href="#hours">continue</a></button>
```

Comment:
I think you have problems with required because you have these buttons without onSubmit. Maybe you know that :) Really tricky because you don't want to submit before the last submit-button. I have no idea, this is just a guess, but maybe try something like changing the button-tags to divs/spans or something? 🤷‍♀️